app/views/entities/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="new-category-nav">
+<%= render "shared/back" %>
+<h3 class="new-category-title">Transactions</h3>
+</div>
+<main class="transactions-container">
+    <% if @entities.empty? %>
+    <div class="empty-category">
+        <h3>You don't have any transactions yet! <%= @category.name %> Category</h3>
+        <%= link_to 'Add a new transaction', new_category_entity_path %>
+    </div>
+<% else %>
+    <div class="total-transactions">
+        <span>
+            <%= @category.name %>
+        </span>
+        <Span>
+            $<%= @entities.inject(0) { |sum, e| sum += e.amount  }%> 
+        </Span>
+    </div>
+    <div class="transactions">
+        <% @entities.each do |entity| %>
+            <div class="transaction">
+                    <%= image_tag(@category.icon || '', class: "category-img") %>
+                <div>
+                    <h2>
+                        <%= entity.name %>
+                    </h2>
+                    <span>
+                        <%= entity.created_at.strftime("%d %b %Y") %>
+                    </span>
+                </div>
+                <div>
+                    <Span>
+                        $<%= entity.amount %> 
+                    </Span>
+                </div>
+
+                    </div>
+                    <% end %>
+                    </div>
+                <%= link_to 'Add new transaction', new_category_entity_path, class: 'add-transaction-btn' %>
```

Comment:
- [ ] [OPTIONAL] the add transaction button should be at the bottom of your page as it is in the design :+1:

Reply:
the main problem was in the UI and some logic, It was not clear in the checkboxes which category you assign a transaction to, I have separated them now clearly, and you should be able to increase the amount accordingly.